mantidproject/mantid
# Muon - ApplyMuonDetectorGrouping Algorithm

Added an algorithm to group detectors within a workspace or workspace group, for use in the Muon Analysis GUI

**To test:**

Open the Muon Analysis Interface (Interfaces->Muon->Muon Analysis), select the EMU instrument and Load the current run. Now open the algorithm Data Handling->Grouping->ApplyMuonDetectorGrouping. The Workspaces should be filled as MuonAnalysis and EMU<current run number>. Call the groupName "test", and the Grouping 1,2,3. Set the plotType to Counts. Leave all the other boxes.

When the algorithm is run, two new workspaces should be created in the EMU<current run number> workspace group. The workspace whose name ends in "_Raw" should have y-values equal to the sum of the first three histograms in the MuonAnalysis workspace. The errors should be added in quadrature.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
